assets/src/js/header.js

```diff
@@ -1,177 +1,146 @@
-export const setupHeader = function($) {
-  'use strict';
+const updateGaAction = (element, elementName) => {
+  element.dataset.gaAction = `${element.ariaExpanded === 'false' ? 'Open' : 'Close'} ${elementName}`;
```

Comment:
I got confused while testing, but this access to aria attributes is [not supported on firefox](https://caniuse.com/?search=ariaExpanded) at the moment :grimacing:  (seems to be [Aria 1.2 recommendations](https://www.w3.org/TR/wai-aria-1.2/#idl-reflection-attribute-values), still recent)
Without it I don't see other ways than the [get/setAttribute](https://w3c.github.io/aria/#idl_example_usage) function that its supposed to replace

Reply:
Oopsie 😬 Updated!